There's linting errors related to the video branch. Can you fix them @chenghsinw? 

```
it-20750:engine-theme-sdk howardj$ npm run lint

> @wpmedia/engine-theme-sdk@2.2.6-canary.1 lint /Users/howardj/sites/engine-theme-sdk
> eslint . --ext .ts,.js,.tsx,.jsx


/Users/howardj/sites/engine-theme-sdk/src/components/Video/index.test.jsx
   7:32  error  Missing semicolon                              semi
   8:5   error  Missing semicolon                              semi
  22:43  error  Missing semicolon                              semi
  38:43  error  Missing semicolon                              semi
  54:43  error  Missing semicolon                              semi
  70:43  error  Missing semicolon                              semi
  73:4   error  Newline required at end of file but not found  eol-last

/Users/howardj/sites/engine-theme-sdk/src/components/Video/index.tsx
   5:33  warning  Unexpected any. Specify a different type                                                                           @typescript-eslint/no-explicit-any
  47:41  error    The closing bracket must be aligned with the line containing the opening tag (expected column 7 on the next line)  react/jsx-closing-bracket-location

✖ 9 problems (8 errors, 1 warning)
  8 errors and 0 warnings potentially fixable with the `--fix` option.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @wpmedia/engine-theme-sdk@2.2.6-canary.1 lint: `eslint . --ext .ts,.js,.tsx,.jsx`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @wpmedia/engine-theme-sdk@2.2.6-canary.1 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/howardj/.npm/_logs/2020-09-24T15_58_11_902Z-debug.log
it-20750:engine-theme-sdk howardj$

--- a/src/components/Video/index.test.jsx
+++ b/src/components/Video/index.test.jsx
@@ -4,8 +4,8 @@ import Video from '.';
 
 describe('Video', () => {
   beforeEach(() => {
-    global.powaBoot = jest.fn()
-  })
+    global.powaBoot = jest.fn();
+  });
 
   describe('with minimum configurations', () => {
     it('set proper data attributes and call window.powaBoot', () => {
@@ -19,7 +19,7 @@ describe('Video', () => {
       expect(videoElement.prop('data-playthrough')).toBe(false);
       expect(videoElement.prop('data-muted')).toBe(false);
       expect(videoElement.prop('data-aspect-ratio')).toBe(0.5625);
-      expect(global.powaBoot).toBeCalled()
+      expect(global.powaBoot).toBeCalled();
     });
   });
 
@@ -35,7 +35,7 @@ describe('Video', () => {
       expect(videoElement.prop('data-playthrough')).toBe(false);
       expect(videoElement.prop('data-muted')).toBe(true);
       expect(videoElement.prop('data-aspect-ratio')).toBe(0.5625);
-      expect(global.powaBoot).toBeCalled()
+      expect(global.powaBoot).toBeCalled();
     });
   });
 
@@ -51,7 +51,7 @@ describe('Video', () => {
       expect(videoElement.prop('data-playthrough')).toBe(true);
       expect(videoElement.prop('data-muted')).toBe(false);
       expect(videoElement.prop('data-aspect-ratio')).toBe(0.5625);
-      expect(global.powaBoot).toBeCalled()
+      expect(global.powaBoot).toBeCalled();
     });
   });
 
@@ -67,7 +67,7 @@ describe('Video', () => {
       expect(videoElement.prop('data-playthrough')).toBe(false);
       expect(videoElement.prop('data-muted')).toBe(false);
       expect(videoElement.prop('data-aspect-ratio')).toBe(0.75);
-      expect(global.powaBoot).toBeCalled()
+      expect(global.powaBoot).toBeCalled();
     });
   });
 });

--- a/src/components/Video/index.tsx
+++ b/src/components/Video/index.tsx
@@ -2,7 +2,7 @@ import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 
 declare global {
-  interface Window { powaBoot?: any }
+  interface Window { powaBoot?: Function }
 }
 
 interface VideoProps {
@@ -44,7 +44,8 @@ const Video: React.FC<VideoProps> = (props) => {
         data-autoplay={autoplay}
         data-playthrough={playthrough}
         data-muted={muted}
-        data-aspect-ratio={aspectRatio} />
+        data-aspect-ratio={aspectRatio}
+      />
     </div>
   );
 };

